StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -508,6 +511,59 @@ extension PaymentSheet {
         /// - Note: This is always called on the main thread.
         public var externalPaymentMethodConfirmHandler: ExternalPaymentMethodConfirmHandler
     }
+
+    /// Configuration for custom payment methods
+    @_spi(CustomPaymentMethodsBeta) public struct CustomPaymentMethodConfiguration {
+
+        /// Defines a custom payment method type that can be displayed in PaymentSheet
+        public struct CustomPaymentMethod {
```

Comment:
I think the name should be `CustomPaymentMethodType` since we typically refer to instances of a "payment method type" as a "payment method"

Reply:
Yeah, agree!